sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_utils/configurations.py

```diff
@@ -151,7 +152,19 @@ def _default_span_processors(configurations):
 
 
 def _default_enable_live_metrics(configurations):
-    configurations.setdefault(ENABLE_LIVE_METRICS_ARG, False)
+    default = False
+    live_metrics_enabled_env_var = environ.get(APPLICATIONINSIGHTS_PREVIEW_LIVE_METRICS_ENABLED, "false").lower()
+    if live_metrics_enabled_env_var == 'true':
+        default = True
+    configurations.setdefault(ENABLE_LIVE_METRICS_ARG, default)
+
+
+def _get_otel_disabled_instrumentations():
```

Comment:
What's this change for?

Reply:
This was a Copy/paste mistake that has been remove.